### DISCUSSION TITLE
Cache issues

### DISCUSSION THREAD

**Discussion by odama626** [directus/directus#5580] [Archive]
I've been running into caching issues, when I create an item for a collection and then fetch the collection instead of returning with the new item it returns a 304 not modified.

Also, disabling the cache is not possible because `cache-control` is not marked as an allowed header in the server for cors

---

### Project details
Directus Version: 9.0.0-rc.62
Environment: production
OS: Linux 4.15.0-142-generic
Node: 14.16.1

**Comment by rijkvanzanten** [comment]
@odama626 Is this a self-hosted instance?

**Reply by odama626** [reply]
yes, it is self hosted. I started with this https://docs.directus.io/guides/installation/docker/#docker-compose

**Reply by rijkvanzanten** [reply]
OK! Take a look at the environment variable options. All of these things are configurable 🙂 https://docs.directus.io/reference/environment-variables/#cors

**Reply by odama626** [reply]
Oh! that is fanstic!

would it make sense to add a note about the default cache purge settings in the docker compose section?

**Reply by rijkvanzanten** [reply]
Nah, I don't think we should duplicate that information between pages. All options including their defaults are listed on that page linked above 👍🏻
